### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by ProPontifex**
See title

### CODE REVIEW

File: streams/us.m3u [hunk 1]
```diff
@@ -95,8 +95,6 @@ https://5aafcc5de91f1.streamlock.net/almagd.tv/almagd.smil/playlist.m3u8
 https://content.uplynk.com/channel/26bd482ffe364a1282bc3df28bd3c21f.m3u8
 #EXTINF:-1 tvg-id="AmericanHorrors.us",American Horrors (480p)
 https://amhorstream.com/playlist/amhor/playlist.m3u8
-#EXTINF:-1 tvg-id="AMGTV.us",AMG TV (1080p)
-https://2-fss-2.streamhoster.com/pl_138/201660-1270634-1/playlist.m3u8
```
Comment (freearhey) [inline streams/us.m3u:99]: working link
image

File: streams/us.m3u [hunk 2]
```diff
@@ -311,8 +301,6 @@ https://cbsta49f-cbsta49f-ms.global.ssl.fastly.net/amagi7b98-AmagiMixible/master
 https://eyeonesports.com/ES2RA-628g.m3u8
 #EXTINF:-1 tvg-id="EternalLifeTVNetwork.us",Eternal Life TV Network (1080p)
 https://livecdn.live247stream.com/eternallife/tv/playlist.m3u8
-#EXTINF:-1 tvg-id="",Everyday Refresh
-https://qvc-amd-live.akamaized.net/hls/live/2034113/lsqvc6us/master.m3u8
```
Comment (freearhey) [inline streams/us.m3u:315]: working link
image

File: streams/us.m3u [hunk 3]
```diff
@@ -827,8 +811,6 @@ https://stream-us-east-1.getpublica.com/playlist.m3u8?network_id=37
 http://api.toonamiaftermath.com:3000/est/playlist.m3u8
 #EXTINF:-1 tvg-id="",Toonami Aftermath Movies (540p)
 http://api.toonamiaftermath.com:3000/movies/playlist.m3u8
-#EXTINF:-1 tvg-id="",Toonami Aftermath Radio (480p)
-http://api.toonamiaftermath.com:3000/radio/playlist.m3u8
```
Comment (freearhey) [inline streams/us.m3u:831]: working link
image

**Review by freearhey — CHANGES_REQUESTED** [review]

File: streams/us.m3u (hunk 2 above)
Comment (R4IN2007) [reply streams/us.m3u:315]: QVC Has news channels for the holidays I guess and some links have change.
Holiday Headquarters (Everyday Refresh)
https://qvc-amd-live.akamaized.net/hls/live/2034113/lsqvc6us/master.m3u8
The Retail Therapy Channel
https://qvc-amd-live.akamaized.net/hls/live/2034113/lsqvc8us/master.m3u8

**Pull request closed without merging**